applications/api-cat/src/test/java/no/acat/restapi/HarvestControllerTest.java

```diff
@@ -0,0 +1,84 @@
+package no.acat.restapi;
+
```

Comment:
HarvestController har to metoder. De to metoder begger gjør bare ett ting. så jeg anbefaler at du har to tester:

triggerHarvestAll_ShouldAddTask
triggerHarvestApiRegistration_ShouldAddTask

mocke addTask.

De tester som du har her akkurat nå, skulle kanskje passe i queue unit test class. eller slettet.